src/rust/python-parser/cxx-bridge.c++

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) 2017-2022 Cloudflare, Inc.
+// Licensed under the Apache 2.0 license found in the LICENSE file or at:
+//     https://opensource.org/licenses/Apache-2.0
+
+// #include <workerd/rust/python-parser/lib.rs.h>
+#include "workerd/rust/cxx-integration/lib.rs.h"
+
+#include <kj/test.h>
```

Comment:
Nit: kj/test.h appears unused here, the commented include above can be dropped too